curp/src/server/curp_node.rs

```diff
@@ -295,19 +362,26 @@ impl<C: 'static + Command> CurpNode<C> {
         let storage_c = Arc::clone(&storage);
         let _ig = tokio::spawn(async move {
             // establish connection with other servers
-            let connects = rpc::connect(others, tx_filter).await;
-            let tick_task = tokio::spawn(Self::tick_task(Arc::clone(&curp_c), connects.clone()));
-            let sync_task = tokio::spawn(Self::sync_task(
-                Arc::clone(&curp_c),
-                connects.clone(),
-                sync_rx,
-            ));
-            let calibrate_task = tokio::spawn(Self::calibrate_task(curp_c, connects, calibrate_rx));
+            let connects = rpc::connect(others.clone(), tx_filter).await;
+            let election_task =
+                tokio::spawn(Self::tick_election(Arc::clone(&curp_c), connects.clone()));
```

Comment:
I suggest renaming tick_election to election_task

Reply:
Fixed